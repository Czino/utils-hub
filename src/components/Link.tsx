export const Link = ({ className, ...props }: React.ComponentProps<'a'>) => (
  <a className={['text-highlight-1 hover:text-highlight-2 active:text-highlight-3', className].join(' ')} {...props} />
)
