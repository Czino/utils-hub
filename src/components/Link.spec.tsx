import { render } from '@testing-library/react'
import { Link } from './Link'

describe('Link', () => {
  it('should render Link', () => {
    const { asFragment } = render(
      <Link className="mt-4" href="/link">
        Test
      </Link>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
