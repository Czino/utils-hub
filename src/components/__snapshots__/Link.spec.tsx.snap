// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render Link 1`] = `
<DocumentFragment>
  <a
    class="text-highlight-1 hover:text-highlight-2 active:text-highlight-3 mt-4"
    href="/link"
  >
    Test
  </a>
</DocumentFragment>
`;
